bitnami/mongodb/templates/arbiter/statefulset.yaml

```diff
@@ -145,11 +145,11 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: K8S_SERVICE_NAME
-              value: "{{ include "mongodb.fullname" . }}-arbiter-headless"
+              value: "{{ include "mongodb.arbiterservicename" . }}"
             - name: MONGODB_REPLICA_SET_MODE
               value: "arbiter"
             - name: MONGODB_INITIAL_PRIMARY_HOST
-              value: "{{ include "mongodb.fullname" . }}-0.{{ include "mongodb.fullname" . }}-headless.$(MY_POD_NAMESPACE).svc.{{ .Values.clusterDomain }}"
+              value: "{{ include "mongodb.fullname" . }}-0.{{ include "mongodb.servicename" . }}.$(MY_POD_NAMESPACE).svc.{{ .Values.clusterDomain }}"
```

Comment:
change servicename to servicenameOverride

Reply:
Fixed